tests/app/celery/test_reporting_tasks.py

```diff
@@ -56,9 +58,12 @@ def test_create_nightly_notification_status_triggers_tasks_for_days(notify_api,
     mock_celery = mocker.patch('app.celery.reporting_tasks.create_nightly_notification_status_for_day')
     create_nightly_notification_status(day_start)
 
-    assert mock_celery.apply_async.call_count == 4
-    for i in range(4):
-        assert mock_celery.apply_async.call_args_list[i][1]['kwargs'] == {'process_day': expected_kwargs[i]}
+    assert mock_celery.apply_async.call_count == 4 * 3  # four days, three notification types
```

Comment:
Nice comment

Reply:
wait till you see my next PR where this got more complicated 😅